lib/fs.js

```diff
@@ -376,6 +376,16 @@ fs.exists = function(path, callback) {
   }
 };
 
+Object.defineProperty(fs.exists, internalUtil.promisify.custom, {
+  enumerable: false,
```

Comment:
I like being explicit in code, especially if it helps clarify intent and when the defaults are not obvious to casual readers, which is why I added the `enumerable: false`; but I won’t force that on you if you don’t like it.

(@dfabulich Thanks for being so quick with updating here, but just so there’s no confusion: Our opinions aren’t inherently better than yours, and if you think a collaborator makes a request that you don’t agree with, you should feel free to say so.)